### ISSUE TITLE
`gu` in JDK11 distribution is invalid

### ISSUE THREAD

**Issue by bobmcwhirter** [oracle/graal#1392]
When building the JDK11-centric distribution, the `bin/gu` still presumes it's under a non-existent JRE.

As it's an extracted dependency (I guess?) I'm finding no simple way to rewrite or provide an alternative `gu_jdk11` source for the script.

**Comment by sdedic** [comment]
@bobmcwhirter are you using `master` sources ? The [launcher script](https://github.com/oracle/graal/blob/master/vm/mx.vm/gu) seems correct (assuming `lib/installer`). Other hardcoded paths seem to be fixed by commit 627e13e900a

could you please elaborate ?

**Comment by bobmcwhirter** [comment]
So yes, I guess the launcher script seems correct, but it's not finding it's own jar:

```
$ ./latest_graalvm_home/bin/gu
Error: Could not find or load main class org.graalvm.component.installer.ComponentInstaller
Caused by: java.lang.ClassNotFoundException: org.graalvm.component.installer.ComponentInstaller
$ ls -l latest_graalvm_home/lib/installer/
total 0
drwxr-xr-x  3 bob  staff  96 Jun 18 09:59 bin
drwxr-xr-x  3 bob  staff  96 Jun 18 09:52 components
$
```

The script seems to assume `lib/installer/*.jar` might exist?

**Comment by bobmcwhirter** [comment]
Working on a fix. javaCompliance being applied more liberally.

**Comment by sdedic** [comment]
Already fixed by 6c1fe58b17144618af14f46ebe4b7eb9c2d83ba6
